spec/mailers/previews/user_mailer_preview.rb

```diff
@@ -1,14 +1,14 @@
 class UserMailerPreview < ActionMailer::Preview
-  def confirm_notifications_sent
-    UserMailer.confirm_notifications_sent(Diagnosis.completed.sample)
-  end
-
   def match_feedback
     feedback = Feedback.all.sample
```

Comment:
```suggestion
    feedback = Feedback.category_need.sample
```
J'ai des preview qui ont planté, sans ça :)